### ISSUE TITLE
Fatal error while generting documentation

### ISSUE THREAD

**Issue by Hackwar** [phpDocumentor/phpDocumentor#3227]
Hi, when generating my documentation, I get this fatal error:

```
CRITICAL  [console] Error thrown while running command "-d './repos/joomla-cms/libraries/src' -t './build/cms-/' --template './' --title 'Joomla! CMS 4.1.x API' --setting='graphs.enabled=true'". Message: "phpDocumentor\Descriptor\Tag\SinceDescriptor::setVersion(): Argument #1 ($version) must be of type string, null given, called in /opt/phpdoc/src/phpDocumentor/Descriptor/Builder/Reflector/Tags/SinceAssembler.php on line 37" ["exception" => TypeError { …},"command" => "-d './repos/joomla-cms/libraries/src' -t './build/cms-/' --template './' --title 'Joomla! CMS 4.1.x API' --setting='graphs.enabled=true'","message" => "phpDocumentor\Descriptor\Tag\SinceDescriptor::setVersion(): Argument #1 ($version) must be of type string, null given, called in /opt/phpdoc/src/phpDocumentor/Descriptor/Builder/Reflector/Tags/SinceAssembler.php on line 37"]
```

In another issue it was said that phpDocumentor would give better debug output, but I can't get it to work properly. Interestingly, we have a very old setup where this worked, but I can't get this to run on our new setup (and can't find out the old version of phpdocumentor) I tried both phpDocumentor 3.3.1, 3.0 and 2.9 and even on different PHP versions, but it always fails with the above error.

**Comment by mvriel** [comment]
I just checked the code and I _think_ that you have a malformed `@since` tag (or one missing the version). I can see in our code that that may result in a null value being passed into a string; for this I will create a PR to fix that in the upcoming release

**Comment by mvriel** [comment]
I have created a pull request to fix this issue. It will be included in the next release, or you can try out the phar file that is made as part of the build pipeline once it is merged

**Comment by Hackwar** [comment]
Thank you!
